### ISSUE TITLE
[Question] Export, import modules

### ISSUE THREAD

**Issue by huytran0605** [feathersjs/feathers#625]
Currently, my project base on feather-cli and i need to use import, export module instead of `require`
But i got the error `SyntaxError: Unexpected token export` when i try to use import, export.
I saw some issue make feather turn back into commonjs.
I understand after https://github.com/feathersjs/feathers/issues/608 approve and merged, i can use import, export without error.
So my question is what can i do to use import & export instead of `require` ?

**Comment by zusamann** [comment]
You could use `babel-register` to bootstrap your code as it runs.
```js
// package.json
}
  "dependencies": {
  "babel-preset-es2015": "^6.24.1",
  "babel-register": "^6.24.1",
  ...
}
```
```js
// .babelrc
{
  "presets": ["es2015"]
}
```
```js
// index.js < main entry point
require('babel-register');
require('./server').run();
```
```js
// server.js < your actual code here
import express from 'express';
export const run () => {
     const app = express()
     ......
}
```
Or you could simply transpile the code using `babel` in case you do not want to use `babel-register` at runtime.

**Comment by daffl** [comment]
The generated application will always only use what the latest version of Node supports (we'll probably update it to use `async/await` for the next version). You can find more of the reasoning behind not using Babel by default in https://github.com/feathersjs/generator-feathers/issues/31 and a discussion in https://github.com/feathersjs/feathers-cli/issues/91 (including the challenges of updating our generator codemods to the new syntax).

In summary, using ES modules with Babel will increase startup/compile time and possibly memory consumption just to use a spec whose implementation details (especially for Node) have not been fully finalized.

If you still want to use it, you can probably run the generated  code through a [require to ES module tranformer](https://github.com/5to6/5to6-codemod) with the Babel setup that @zusamann pointed out.
